python/grass/jupyter/__init__.py

```diff
@@ -5,20 +5,38 @@
 # PURPOSE:   Display classes and setup functions for running GRASS GIS
 #            in Jupyter Notebooks
 #
-# COPYRIGHT: (C) 2021 Caitlin Haedrich, and by the GRASS Development Team
+# COPYRIGHT: (C) 2021-2022 Caitlin Haedrich, and by the GRASS Development Team
```

Comment:
Feel free to add your name here (and @petrasovaa)

Reply:
Done. Added to AUTHORS section, but it might be better to have authors in the documentation and here keep only the file authors. We don't really have a system for that, but something to consider.